shared/metrics/src/test/java/io/pravega/shared/metrics/StatsLoggerProxyTest.java

```diff
@@ -139,14 +139,18 @@ private TestMetric create(String name) {
         }
     }
 
+    //TODO: get rid of this class.
```

Comment:
So, are you going to get rid of it or not?

Reply:
I'm actively working on the @TODOs, just feel it's easier for the team to review with the minimum essential changes first, and then gradually go with further changes.

Reply:
Fixed.